### ISSUE TITLE
Encourage PGP

### ISSUE THREAD

**Issue by chreekat** [hackercouch/hackercouch#75]
I just put a note about my key in my "profile" section, but it would be great if it was a supported piece of metadata.

**Comment by captn3m0** [comment]
Even if we do, it won't be treated specially unless we decide to fetch keys from public servers (which might fail as well). Would the following work for you:

`pgp: 0x20068bfb`

YAML supports hexadecimal, so this should be fine.

**Comment by chreekat** [comment]
Yeah I think that's all I was really expecting. :) I haven't looked at the
code or website too much to have ideas for how to show that on the UI,
though.

If keybase.io were a bigger thing, I suppose an alternative would be to
allow `keybase: chreekat` and link appropriately.

**Comment by simison** [comment]
@chreekat adding new fields is pretty straightforward, these two files:
https://github.com/hackercouch/hackercouch/blob/gh-pages/_layouts/couch.html
https://github.com/hackercouch/hackercouch/blob/gh-pages/add.md

**Comment by chreekat** [comment]
Cool, I'll poke at it later. :)
